.github/RELEASE_GUIDE.md

```diff
@@ -0,0 +1,50 @@
+# NMDC_NOTEBOOKS Release Guide
+
+## Introduction
+The following is a set of guidelines for generating releases for nmdc_notebooks. This documentation is solely for the maintainers of the nmdc_notebooks repo. A release is generated on the first business day of every month if there have been pull requests merged into main. If there are no changes on main, then no release is generated.
+
+## Preparing for Release
+
+### Versioning Scheme
```

Comment:
This is great.  One minor improvement would be to make sure each bullet is consistent stylistically.  Some start with capitalization, others do not.  Some end it periods, others do not.